lib/webdriverLogger.ts

```diff
@@ -0,0 +1,55 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import * as stream from 'stream';
+
+import {CommandName, WebDriverCommand} from './webdriverCommands';
+
+export class WebDriverLogger {
+  logStream: stream.Writable;
+
+  public setLogDir(logDir: string) {
+    let logName = `webdriver_log_${process.pid}.txt`;
```

Comment:
I thought of that, but when protractor runs with multicapabilities we'll be starting several proxies at the same time. We could put pid and timestamp in the name, or if that's too verbose maybe a short identifier that's a hash of both the pid and timestamp?